### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by mwanawabangona**
- Add Html
- Add Css
- Add Linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -309,6 +309,76 @@ <h2 class="heading-primary">About Myself</h2>
         </div>
```
Comment (SajjadAhmad14) [inline index.html:309]: Please add all unnecessary files and folders like `.vscode` to `.gitignore`.

File: index.html [hunk 2]
```diff
@@ -309,6 +309,76 @@ <h2 class="heading-primary">About Myself</h2>
...
+      />
+      <div class="contact-text">
+        <h3 class="heading-tertiary">Contact me</h3>
+        <p class="secondary-text">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <form
```
Comment (SajjadAhmad14) [inline index.html:327]: Please add image decorations spotted in the screenshot in the right place of the form.
images
.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Ellis Ng'ona, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (mwanawabangona) [reply index.html:309]: OK noted

File: index.html (hunk 2 above)
Comment (mwanawabangona) [reply index.html:327]: Done

**Review by AshakaE — APPROVED** [review]
### Hi @mwanawabangona  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**